Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/setup_unittest.py

```diff
@@ -89,7 +87,7 @@ def test_git(self):
         self.assertEqual(
             captured.stdout.getvalue(),
             'For detailed information about options this script is configuring see:\n'
```

Comment:
This is slightly twisted to read. Might re-word something like: 
'For detailed information about the options configured by this script, please see:\n'